localstack/services/install.py

```diff
@@ -273,6 +278,13 @@ def install_all_components():
     # install all components
     install_components(DEFAULT_SERVICE_PORTS.keys())
 
+def install_debugpy_and_dependencies():
+    try:
+        import debugpy
+    except:
+        run('apk add %s' %  (" ".join(DEBUGPY_DEPENDENCIES)) )
```

Comment:
Thanks for updating the PR @pinzon . I think this is a neat approach, installing the dependencies on demand.

My only suggestion here would be to prefetch and cache the packages via `apk fetch ...`, instead of downloading them every time. We can download them to a pre-defined location inside `config.TMPDIR`, which should get persisted across restarts of LocalStack (i.e., usually this folder is mounted into the container). This would help us optimize the network traffic, and speed up the startup time a bit more.. 👍 Thanks!